src/notebooks/filesystem-tutorial.ipynb

```diff
@@ -275,35 +288,43 @@
        "      <td>604938</td>\n",
        "      <td>Roland Phillips</td>\n",
        "      <td>Roland.Phillips@studentgps.org</td>\n",
-       "      <td>Active</td>\n",
-       "      <td>8/20/2020 12:34</td>\n",
-       "      <td>8/20/2020 12:34</td>\n",
+       "      <td>2020-08-20 12:34:56</td>\n",
+       "      <td>2020-08-20 12:34:56</td>\n",
+       "      <td>NaT</td>\n",
+       "      <td>NaT</td>\n",
        "    </tr>\n",
        "  </tbody>\n",
        "</table>\n",
        "</div>"
       ],
       "text/plain": [
-       "   SourceSystemIdentifier SourceSystem UserRole LocalUserIdentifier  \\\n",
-       "0               100032890    Schoology  student         mary.archer   \n",
-       "1               100032891    Schoology  student         kyle.hughes   \n",
-       "2               100032892    Schoology  student          peter.nash   \n",
-       "3               100032893    Schoology  student       larry.mahoney   \n",
-       "4               100032894    Schoology  student     roland.phillips   \n",
+       "  SourceSystemIdentifier SourceSystem UserRole LocalUserIdentifier  \\\n",
```

Comment:
In my recent changes, all of the sample files now have "BestLMS" as the `SourceSystem` instead of "Schoology" or "Canvas". You might need to rebase.

Reply:
Rebased ✔️